src/engraving/dom/score.h

```diff
@@ -997,8 +997,8 @@ class Score : public EngravingObject, public muse::Injectable
     std::vector<Lyrics*> lyrics() const;
     String extractLyrics() const;
 
-    int keysig();
-    int duration();
+    int keysig() const;
+    int duration() const;
     int durationWithoutRepeats();
```

Comment:
Could be made const as well, while at it

Reply:
added const